freedomofpress/securedrop
# fix restore logic to recreation of onion services

## Status

Ready for review

## Description of Changes

Fixes #3960

Alter the restore logic to ensure that the onion services from the backup are what end up running on prod, and not the new values that may have been generated by Ansible.

## Testing

- Provision 0.11.1 machine (trusty)
- Run backup script
- Provision 0.12.0 machine (xenial)
- Run restore script
- Ensure restore script exist successfully
- Ensure tor ssh access to machine using values from 0.11.1 backup
- Ensure web app "just works"

## Deployment

This does not change deployment, but it does change restore from a previous deployment. The admin will run the same commands on xenial as on trusty.
